packages/fiber/src/three-types.ts

```diff
@@ -450,7 +453,19 @@ export interface ThreeElements {
   shape: ShapeProps
 }
 
-declare global {
```

Comment:
I think we'll need to keep this for v8 (master) since some user-land code references global `JSX`. We have a v9 branch dedicated to React 19 where we can remove this. It's currently in alpha to enter beta this week.